config/nunjucks/globals.js

```diff
@@ -8,18 +8,27 @@ const {
   FEEDBACK_URL,
   SUPPORT_EMAIL,
   FEATURE_FLAGS,
+  IS_PRODUCTION,
 } = require('../')
 const { mountpath: componentsUrl } = require('../../app/components')
 const i18n = require('../i18n')
 const logger = require('../logger')
 const { manifest: manifestPath } = require('../paths')
 
 let webpackManifest = {}
+
+const supportLink = IS_PRODUCTION
+  ? 'https://support.hmpps.service.justice.gov.uk/feedback-and-support/book-secure-move'
+  : 'https://support-dev.hmpps.service.justice.gov.uk/feedback-and-support/book-secure-move'
```

Comment:
I'm thinking we should move this logic in to the `config/index.js` file, in a similar way to how we have `FEEDBACK_URL` coming directly from the config.

It's not a big deal, but it means we can use this in multiple places if we needed to without needing to extra the logic out of this file.

Reply:
Good shout